frontend/utils.js

```diff
@@ -17,6 +17,23 @@ export const formatBytes = bytes => {
     return `${Math.max(bytes, 0.1).toFixed(1)} ${units[i]}`;
 };
 
+export const formatTimestamp = stamp => {
+    if (stamp <= 0) {
+        return '';
+    }
+
+    const date = new Date(stamp * 1000);
+
+    return new Intl.DateTimeFormat('de-DE', {
```

Comment:
why is it always 'de-DE'? maybe use 'default'?

Reply:
Fixed